Pod/Classes/SeafUploadFile.m

```diff
@@ -568,28 +568,41 @@ - (NSURL *)previewItemURL
     return _preViewURL;
 }
 
-- (UIImage *)icon
-{
-    if (_asset)
-        return [UIImage imageWithCGImage:_asset.thumbnail];
-
-    UIImage *img = [self isImageFile] ? self.image : nil;
-    return img ? [Utils reSizeImage:img toSquare:32.0f] : [UIImage imageForMimeType:self.mime ext:self.name.pathExtension.lowercaseString];
+- (UIImage *)icon {
```

Comment:
Is it worth caching the icon?
As the sync mode is used, it might take a long time to load the image from PHImageManager